lib/config/env.dart

```diff
@@ -75,8 +76,8 @@ class Env {
   static const String extrinsicSuccessEvent = "ExtrinsicSuccess";
   static const String extrinsicFailedEvent = "ExtrinsicFailed";
   static const String approved = "Approved";
-  static const String peaqTestnet = "wss://wss.agung.peaq.network";
-  // static const String peaqTestnet = "ws://10.0.2.2:9944";
+  // static const String peaqTestnet = "wss://wss.agung.peaq.network";
+  static const String peaqTestnet = "ws://10.0.2.2:9944";
```

Comment:
Should we change to the Agung network?

Reply:
Sorry, I forgot to change it back to production url. 
Thanks for the catch.

Reply:
Fixed